CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## Solidus (not yet versioned)
+
+*   Address is immutable (Address#readonly? is always true)
+
+    This allows us to minimize cloning addresses, while still ensuring historical
+    data is preserved.
+
+*   PUT /api/addresses without an order no longer works
```

Comment:
Also, would it be worth adding a changelog entry saying that admin user search now searches all user addresses instead of just the current defaults?